docs/lumi.rst

```diff
@@ -45,19 +45,16 @@ We set the compilers to the cray compiler wrappers (which in our case use ``amdc
 
 .. code-block:: bash
 
-    module load LUMI/23.09 partition/G
-    module load cpeAMD/23.09
-    module load rocm/5.6.1
-    module load amd/5.6.1
-
-    module load buildtools/23.09
-
-    module load Boost/1.82.0-cpeAMD-23.09
+    module load LUMI/24.03 partition/G
+    module load cpeAMD/24.03
+    module load rocm/6.0.3
+    module load amd/6.0.3
+    module load Boost/1.83.0-cpeAMD-24.03
```

Comment:
Should only be needed for an ACPP installation (which has become much more difficult starting with ROCm 6.0 and beyond; since libLLVM isn't bundled with ROCm anymore since then)